app/views/submissions/_assignment_files_info.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class='pane-wrapper'>
+  <div class='pane' style='margin-bottom: 1em;'>
```

Comment:
Hmm, is the style here necessary for some spacing? This is a child of the outer `div`, so even if it was adding some space, it would probably be better to include it on the parent `div`.